ios/Podfile

```diff
@@ -75,6 +75,12 @@ abstract_target 'Status' do
         # TODO : remove after upgrading to react-native 0.72.x
         config.build_settings['GCC_PREPROCESSOR_DEFINITIONS'] ||= ['$(inherited)', '_LIBCPP_ENABLE_CXX17_REMOVED_UNARY_BINARY_FUNCTION']
       end
+      case target.name
+      when 'ReactNativeCameraKit'
+        target.build_configurations.each do |config|
+          config.build_settings.delete 'IPHONEOS_DEPLOYMENT_TARGET'
+        end
+      end
```

Comment:
Needed to remove predefinition of iOS minimum deployment target for RNCameraKit

Reply:
Hi @ibrkhalil : Interesting approach. 
Ideally when modifying Podfile of a lib we prefer using a `patchPhase` in nix 
example : https://github.com/status-im/status-mobile/blob/develop/nix/deps/nodejs-patched/default.nix#L92-L99

Reply:
Do you think that matching a string in a file and then doing a replacement is better or accessing an attribute in a nested structure and doing a replacement?

I prefer the latter.

But if that's the team's preference happy to do it :)

Reply:
So I should loop through the inner pods of ReactNativeCameraKit and remove their minimum target uses this way, Okay.

Reply:
its fine to do this in podfile as well! I have no strong opinion :)

Reply:
I think Podfile change is less fragile than substituting strings.